source/_docs/drupal-redis.md

```diff
@@ -58,12 +58,15 @@ Enable Redis cache server from your Pantheon Site Dashboard by going to **Settin
       $settings['redis.connection']['password']  = $_ENV['CACHE_PASSWORD'];
 
       $settings['cache']['default'] = 'cache.backend.redis'; // Use Redis as the default cache.
-      $settings['cache_prefix']['default'] = 'pantheon-redis';
+      $settings['cache_prefix']['default'] = VERSION . ':pantheon-redis'; // Version prefix to avoid bugs when upgrading

```

Comment:
@alexdicianu it turns out that it will cause an error when installing a module, i'll be taking this out. 

Agree to this leaving 'pantheon-redis' prefix and for the issue about issue during version upgrade, would just suggest to clear and rebuild cache here for redis